packages/ui/cypress/tests/setIdentity.cy.ts

```diff
@@ -0,0 +1,105 @@
+import { InjectedAccountWitMnemonic } from '../fixtures/injectedAccounts'
+import { landingPageUrl } from '../fixtures/landingData'
+import { setIdentityMultisigs } from '../fixtures/setIdentity/setIdentityMultisigs'
+import { setIdentitySignatories } from '../fixtures/setIdentity/setIdentitySignatories'
+import { multisigPage } from '../support/page-objects/multisigPage'
+import { sendTxModal } from '../support/page-objects/sendTxModal'
+import { topMenuItems } from '../support/page-objects/topMenuItems'
+import { clickOnConnect } from '../utils/clickOnConnect'
+import { waitForTxRequest } from '../utils/waitForTxRequests'
+
+const initAndConnect = (account: InjectedAccountWitMnemonic) => {
+  cy.visit(landingPageUrl)
+  cy.initExtension([account])
+  clickOnConnect()
+  cy.connectAccounts([account.address])
+}
+
+describe('Set an identity', () => {
+  it('Can set an identity from the dropdown', () => {
+    const multisigSignatoryWithoutIdentity = setIdentitySignatories[1]
+    initAndConnect(multisigSignatoryWithoutIdentity)
+    multisigPage.optionsMenuButton().click()
+    multisigPage.setIdentityMenuOption().should('be.visible').click()
+    sendTxModal.sendTxTitle().should('be.visible')
+    sendTxModal.setIdentitySection().should('be.visible')
+
+    // every field is empty to start with
+    sendTxModal.setIdentityFieldInput('display').should('have.value', '')
+    sendTxModal.setIdentityFieldInput('legal').should('have.value', '')
+
+    // typing into legal with empty display should show an error
+    sendTxModal.setIdentityField('legal').type('le')
+    sendTxModal.sendTxError().should('contain', 'Display name is required')
+    sendTxModal.setIdentityFieldElement('display', 'label').should('havInpute.class', 'Mui-error')
```

Comment:
My approach to this would have been to put a locator on the error itself then in test yield the parent element of the applicable field input, use cy.within() to narrow the scope, and then search for the error element inside there, you probably wouldn't need setIdentityField then. 

This is all very readable though so happy to keep it like it is.